src/sentry/api/serializers/models/incident.py

```diff
@@ -56,3 +58,18 @@ def serialize(self, obj, attrs, user):
             'totalEvents': aggregates['count'],
             'uniqueUsers': aggregates['unique_users'],
         }
+
+
+class DetailedIncidentSerializer(IncidentSerializer):
```

Comment:
@lynnagara We should probably implement `get_attrs` here so that it'll be efficient if we serialize more than one Incident with this.